samples/SmokeTest/src/main/java/com/azure/CosmosDB.java

```diff
@@ -0,0 +1,127 @@
+package com.azure;
+
+import com.azure.data.cosmos.CosmosClient;
+import com.azure.data.cosmos.CosmosContainer;
+import com.azure.data.cosmos.CosmosItemProperties;
+import com.azure.data.cosmos.FeedResponse;
+import com.azure.data.cosmos.FeedOptions;
+import reactor.core.publisher.Flux;
+import reactor.core.scheduler.Schedulers;
+
+public class CosmosDB {
+    private static CosmosClient client;
```

Comment:
These don't need to be static variables, they can be passed as parameters to other methods.